elections/kubeflow-steering-committee-elections-2023.md

```diff
@@ -1 +1,92 @@
-This file will contain guidelines, overview, timeline, phases, list of eligible voters, list of eligible candidates and results from the election.
+# Kubeflow Project Steering Committee Elections 2023 #
+## Guidelines, Timelines and Phases ##
+
+The Kubeflow Project is holding its first election for the Kubeflow Project Steering Committee (PSC).  The PSC will also be responsible for helping to setting up of Technical Oversight Committee (TOC) that is responsible for defining and maintaining the technical vision of the Kubeflow Project and an Administrative Oversight Committee (AOC) that is responsible for defining and maintaining the outreach, growth and other non-code specific items of the Kubeflow project. The TOC and AOC will be set up at a future date and will have its own defining role in the Kubeflow project.  
+
+The PSC responsibilities and operations are defined [here](https://github.com/kubeflow/community/blob/master/proposals/STEERING-COMMITTEE.md). The PSC is intended to be a resource in the community that both guides the direction of the Kubeflow Project and acts as a resource to the TOC, AOC, Working Groups (WG), Special Interest Groups (SIG) and Teams. Their role is also to help define what new WGs, SIGs and Teams. 
```

Comment:
KSC responsibilities: probably better to refer to https://github.com/kubeflow/community/pull/650. There are still a few of issues to close.

Reply:
Ok - I'll just leave it at the here comment.